ui/apps/platform/src/Containers/SystemHealth/DashboardPage.js

```diff
@@ -3,11 +3,12 @@ import { Flex, FlexItem, Grid, GridItem, PageSection, Title } from '@patternfly/
 
 import useInterval from 'hooks/useInterval';
 
+import CertificateCard from './CertificateHealth/CertificateCard';
 import ClustersHealthCards from './ClustersHealth/ClustersHealthCards';
+import DeclarativeConfigurationHealthCard from './DeclarativeConfigurationHealth/DeclarativeConfigurationHealthCard';
+import GenerateDiagnosticBundle from './DiagnosticBundle/GenerateDiagnosticBundle';
+import VulnerabilityDefinitionsHealthCard from './VulnerabilityDefinitionsHealth/VulnerabilityDefinitionsHealthCard';
 
-import DeclarativeConfigurationHealthCard from './Components/DeclarativeConfigurationHealthCard';
-import GenerateDiagnosticBundle from './Components/GenerateDiagnosticBundle';
-import VulnerabilityDefinitionsHealthCard from './Components/VulnerabilityDefinitionsHealthCard';
```

Comment:
@ivan-degtiarenko
1. You might need to resolve merge conflict here when you rebase #6222
2. As we discussed on the call:
    * Please add your rewritten components into IntegrationHealth folder and delete Components folder
    * Also move anything still useful from utils into IntegrationHealth folder and delete utils folder